frontend/src/global.d.ts

```diff
@@ -7,3 +7,8 @@ declare module "*.svg" {
     const Component: React.FC<React.SVGProps<SVGSVGElement>>;
     export default Component;
 }
+
+declare module "*.css" {
+    const value: any;
+    export default value;
+}
```

Comment:
Do we want to look into alternatives to `css-loader` that would give us proper types? :thinking:

Reply:
Ehm... do you want to? I don't see the benefit right now.

Reply:
Yes, I want to, and I did: #180.

The benefit, as always with proper types, is that we find bugs. :wink: